app/generated_resources.grd

```diff
@@ -5937,7 +5937,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
       </message>
       <if expr="use_titlecase">
```

Comment:
I don't think this is quite the right fix, as this string is meant to be used when `use_titlecase` is in effect (for instance, I think in Mac menu items). Instead, I think this string should remain as-is and we'll need to break out the settings item into its own string that doesn't use title case at all.

Reply:
Yes, I wasn't sure about it either. Thought I could give it a try after discussing the matter with Stephen (https://github.com/brave/brave-browser/issues/16939#issuecomment-1144058368), but it's all right if you decide to close this one as I'm not aware how to resolve this issue if the change I proposed is wrong. I'm really interested to see how it's fixed properly 👍

Reply:
No worries! Your proposed fix was my first thought too; I just happened to notice the `use_titlecase` directive above it.